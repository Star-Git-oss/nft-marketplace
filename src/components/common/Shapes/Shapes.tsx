import styled from 'styled-components';
import colors from '../ui-library/styles/colors';

export const RoundedSpan = styled.span`
  border: solid 1px ${colors.black};
  padding: 15px;
  border-radius: 1000px;

`;

export const SquaredSpan = styled.span`
  border: solid 1px ${colors.black};
  padding: 15px;
  border-radius: 5px;
`;