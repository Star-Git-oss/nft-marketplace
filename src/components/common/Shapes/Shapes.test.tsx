/* eslint-disable no-console */
import React from 'react';
import renderer from 'react-test-renderer';
import { RoundedSpan, SquaredSpan } from './Shapes';


// Known issue: error and warning does not work for console
// https://github.com/facebook/react/issues/7047
const original = console.error;

beforeEach(() => {
  console.error = jest.fn();
});

afterEach(() => {
  console.error = original;
});

describe('Shapes snapshots', () => {

  test('RoundedSpan renders correctly', () => {
    const tree = renderer
      .create(<RoundedSpan />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('SquaredSpan renders correctly', () => {
    const tree = renderer
      .create(<SquaredSpan />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

});