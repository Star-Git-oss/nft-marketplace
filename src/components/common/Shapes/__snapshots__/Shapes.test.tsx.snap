// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shapes snapshots RoundedSpan renders correctly 1`] = `
<span
  className="sc-bdfBwQ clAelO"
/>
`;

exports[`Shapes snapshots SquaredSpan renders correctly 1`] = `
<span
  className="sc-gsTCUz krigDU"
/>
`;
